modules/test/playwright/tests/frontend-data-set-admin-web/tests/data-set-admin/systemDataSets.spec.ts

```diff
@@ -9,22 +9,25 @@ import {featureFlagsTest} from '../../../../fixtures/featureFlagsTest';
 import {loginTest} from '../../../../fixtures/loginTest';
 import {waitForAlert} from '../../../../utils/waitForAlert';
 import {dataSetManagerApiHelpersTest} from '../../fixtures/dataSetManagerApiHelpersTest';
+import {actionsPageTest} from './fixtures/actionsPageTest';
 import {systemDataSetsPageTest} from './fixtures/systemDataSetsPageTest';
 
 export const test = mergeTests(
+	actionsPageTest,
 	dataSetManagerApiHelpersTest,
 	systemDataSetsPageTest,
 	featureFlagsTest({
 		'LPD-37531': {enabled: true},
 		'LPS-164563': {enabled: true},
+		'LPS-193005': {enabled: true},
```

Comment:
Will not be needed after https://github.com/liferay-frontend/liferay-portal/pull/4654, but it's good to add new tests with this FF turned on.